### ISSUE TITLE
mkvirtualenv does not support -a , -i , -r options

### ISSUE THREAD

**Issue by dswhite42** [davidmarble/virtualenvwrapper-win#60]
The Linux version of mkvirtualenv has 3 options in addition to those that get passed through to to virtualenv.  virtualenvwrapper-win doesn't not currently support these.  Can they be added?  

```
Usage: mkvirtualenv [-a project_path] [-i package] [-r requirements_file]
                    [virtualenv options] env_name

 -a project_path

    Provide a full path to a project directory to associate with
    the new environment.

 -i package

    Install a package after the environment is created.
    This option may be repeated.

 -r requirements_file

    Provide a pip requirements file to install a base set of packages
    into the new environment.
```

**Comment by fuzzmz** [comment]
Implemented the `-a project_path` option. Going to also take a swing at the `-i` and `-r` flags.

**Comment by thebjorn** [comment]
This is implemented in PR #82.
